docker/development.yml

```diff
@@ -76,6 +76,10 @@ services:
   elasticsearch:
     ports:
       - '9200:9200'
+    ulimits:
```

Comment:
For reference: https://www.elastic.co/guide/en/elasticsearch/reference/7.4/docker.html#_disable_swapping